libs/core/src/index.ts

```diff
@@ -7,4 +7,5 @@ export { shuffle } from './utils/shuffle';
 export { randomSubset } from './utils/random-subset';
 export { randomSlice } from './utils/random-slice';
 
-export * from './storage/public_api';
\ No newline at end of file
+export * from './storage/public_api';
+export * from './magento';
```

Comment:
I think `drivers` is a way better place to put this than `core`.

Reply:
So `driver/src/magento`? We would need to add a testing package to `driver`. FWIW I just realized that this isn't required for the `MagentoProduct` factory.

Reply:
yea, you'd need to add the testing subpackage

Reply:
done